src/main/java/javax/resource/spi/ActivationSpec.java

```diff
@@ -34,7 +34,7 @@
 	 * validation. The implementation of this self-validation check behavior is
 	 * optional.
 	 * 
-	 * Note: Since Java EE Connector 1.6 specification, resource adapter
+	 * Note: Since JavaEE&trade; Connector 1.6 specification, resource adapter
```

Comment:
This is definitely wrong.  Either leave it as is if it's a historical reference, or change it to "Jakarta Connectors".

Reply:
I put back the original wording